File: andtool/src/main/java/com/ofx/andtool/Logg.java
package com.ofx.andtool;

import android.app.ActivityManager;
import android.util.Log;

/**
 * Created by Ofx on 2018/1/6.
 */

public class Logg{
    private final static String default_tag = "show_tag";
     public static boolean isDebug = true;

     public static void e(String msg){
         if (isDebug){
             Log.e(default_tag,msg);
         }
     }

    public static void i(String msg){
        if (isDebug){
            Log.i(default_tag,msg);
        }
    }

}

File: app/src/main/java/com/ofx/show/tool/Logg.kt
package com.ofx.show.tool

import android.util.Log

/**
 * Created by Ofx on 2017/11/28.
 */
object Logg {
    var default_tag = "show_tag"
    var isDebug:Boolean = true;
    fun e(msg:String) {
        if (isDebug) {
            Log.e(default_tag,msg)
        }
    }

    fun i(msg:String) {
        if (isDebug) {
            Log.i(default_tag,msg)
        }
    }
}